gcn_classic_to_json/notices/CALET_GBM_FLT_LC/__init__.py

```diff
@@ -0,0 +1,40 @@
+from ... import utils
+import numpy as np
+
+# Will have to rename schema after the location is decided
+
+
+def parse(bin_arr):
+    # e_high and e_low are both encoded in the same 4 byte integer (as 2 byte shorts)
+    energy_range_data = bin_arr[17:18].view(">i2")
+    # similarly we do this with lat and long
+    lat_lon_data = np.flip(bin_arr[16:17].view(">i2"))
+    lat, lon = lat_lon_data[1], lat_lon_data[0]
+    # Using np.unpackbits to get detector status from bits
+    # I have defined 0 in the binary as "on" and 1 as "triggered"
+    detector_options = ["on", "triggered"]
+    # First convert to an array of 4 1 byte integer values and then convert it into an array of bits
+    # Reversing array so 0th bit is 0-indexed
+    detectors_bits = np.flip(np.unpackbits(bin_arr[19:20].view(dtype="u1")))[:3]
+    detectors_status = [detector_options[bit] for bit in detectors_bits]
+    detectors = dict(zip(["HXM1", "HMX2", "SGM"], detectors_status))
+
+    return {
+        # "$schema" : "",
+        "mission": "CALET",
+        "instrument": "GBM",
+        # "alert_type" : "initial", # does not exist in the binaries
+        # "record_number" = 1, # does not exist in binaries
+        "id": [bin_arr[4]],
+        "messenger": "EM",
+        "trigger_time": utils.tjd_to_jd(bin_arr[5], bin_arr[6]),
+        "trigger_type": "rate",
+        "rate_energy_range": [energy_range_data[1], energy_range_data[0]],
+        "rate_snr": bin_arr[9] * 1e-2,
+        "rate_duration": bin_arr[11] * 1e-2,
```

Comment:
The documentation calls this field `bkg_dur1`. Field 10 is `fore_dur`. Which one corresponds to `rate_duration`?

Reply:
The reason I chose bkg_dur1 is that the documentation defines it as "the amount of time (units centi-sec) used to determine the background countrate when used for the lc_significance" and the descriptions for rate_duration is "Interval over rate signal to noise ratio calculation". However, fore_dur is defined as "the amount of time (in units of centi-sec) the defines the triggering interval", so should I switch to that?

Reply:
@jracusin, @Vidushi-GitHub, would you please check this?

Reply:
None of it corresponds to rate_duration, as that's defined as signal to noise ratio. CALET has defined "bkg_dur1" as interval before the trigger, fore_dur as triggering interval.

Reply:
The documentation defines lc_signif as "signal-to-noise ratio (in units of centi-sigma) during the foreground interval". Are foreground interval and triggerring interval different?